File: library/src/main/java/xyz/fcampbell/rxgms/observables/activity/BaseActivityObservable.java
package xyz.fcampbell.rxgms.observables.activity;

import android.content.Context;

import com.google.android.gms.location.ActivityRecognition;

import xyz.fcampbell.rxgms.observables.BaseObservable;

public abstract class BaseActivityObservable<T> extends BaseObservable<T> {
    protected BaseActivityObservable(Context ctx) {
        super(ctx, ActivityRecognition.API);
    }
}

File: library/src/main/kotlin/xyz/fcampbell/rxgms/observables/activity/BaseActivityObservable.kt
package xyz.fcampbell.rxgms.observables.activity

import android.content.Context

import com.google.android.gms.location.ActivityRecognition

import xyz.fcampbell.rxgms.observables.BaseObservable

abstract class BaseActivityObservable<T> protected constructor(
        ctx: Context
) : BaseObservable<T>(ctx, ActivityRecognition.API)
